src/api/handlers/action.py

```diff
@@ -133,7 +133,7 @@ def update(self, request):
       .expect("action_id", int, is_required=True)
       .expect("title", str, is_required=False, options={"min_length": 4, "max_length": 40})
       .expect("calculator_action", int, is_required=False)
-      .expect("image", "file", is_required=False, options={"is_logo": True})
+      .expect("image", "str_list", is_required=False, options={"is_logo": True})
```

Comment:
with this are you sure things will not go kaboom? 😂 
Are we first uploading the images somewhere and getting links?  Can the links expire?